.gitignore

```diff
@@ -1,2 +1,19 @@
 *.jl.cov
+*.jl.*.cov
 *.jl.mem
+
+.DS_Store
+
+docs/build/
+docs/site/
+
+docs/Manifest.toml
+
+*.ipynb_checkpoints
+**/*.ipynb_checkpoints
+**/**/*.ipynb_checkpoints
+
+_*.dat
+*.swp
+__pycache__/
+Manifest.toml
```

Comment:
I think it's ok to include these, actually. In future we'll be using them for CI anyway.